src/components/QuestionList/QuestionList.vue

```diff
@@ -216,6 +217,7 @@ export default {
   },
   created() {
     // generate user id
+
```

Comment:
y?

Reply:
will be removed when we finished the login sessions.